mewarpx/mewarpx/util.py

```diff
@@ -5,9 +5,26 @@
 import inspect
 import os
 
+import numpy as np
+import minerva.util as minutil
```

Comment:
I believe this will already be fixed in @nruof's PR but it would be good to confirm that we no longer need `minerva` once you merge in his changes (might not be on github yet).

Reply:
Yes